### PR TITLE
generate pages for authors

### PR DESCRIPTION
**Pull request by pjazdzewski1990**

### CODE REVIEW

File: _plugins/authors_gen.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      
+      print "#all:"
+      puts site.posts.length
+      
+      grouped = site.posts.group_by { |post| post.data['author'] }
+      
+      grouped.each { |author, posts|
+        print "Generating for " 
+        print author 
+        print " #"
```
Comment (marioosh) [inline _plugins/authors_gen.rb]: such debug infos not needed after merge.

File: _plugins/authors_gen.rb [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      
+      grouped = site.posts.group_by { |post| post.data['author'] }
+      
+      grouped.each { |author, posts|
+        print "Generating for " 
+        print author 
+        print " #" 
+        puts posts.length  
+        
+        create_author_page(site, author.to_s + ".html", posts, "Posts by " + author.to_s)
```
Comment (marioosh) [inline _plugins/authors_gen.rb]: I'm not sure `autor.to_s` and used in template ` escape_uri` will give you the same result in all cases.

Comment (pjazdzewski1990) [inline _plugins/authors_gen.rb]: @marioosh I'm not sure about that either ... 
Do we need to escape it? I mean the view (via uri_escape) translates "Nosiński" into ... "Nosiński". That's the reason why I didn't escape it in the script. If we change it here we will need another way for the view, too.  
How should I use http://www.rubydoc.info/gems/liquid/Liquid/StandardFilters in this code? It keeps throwing `NoMethod` errors 
Also take a look at http://stackoverflow.com/questions/6625035/utf-8-characters-in-urls

Comment (marioosh) [inline _plugins/authors_gen.rb]: It's ok for me if current solution works with polish letters.

**Pull request merged**